cli/sub_gen.go

```diff
@@ -258,14 +255,14 @@ func MevBoostSubCmd(sedgeAction actions.SedgeActions) *cobra.Command {
 		Long:  "Generate a docker-compose and an environment file with a mev-boost node configuration",
 		Args:  cobra.NoArgs,
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			return preValidationGenerateCmd(network, logging)
+			return preValidationGenerateCmd(network, logging, &flags)
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return runGenCmd(cmd.OutOrStdout(), &flags, sedgeAction, []string{mevBoost})
 		},
 	}
 	// Bind flags
-	cmd.Flags().StringVar(&flags.relayURL, "relay-url", "", "Relay URL used to connect to mev relay.")
+	cmd.Flags().StringSliceVar(&flags.relayURLs, "relay-urls", []string{}, "List of comma separated relay URLs used to connect to mev relay. Example: 'sedge generate mevboost --relay-url=https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net,https://0xa1559ace749633b997cb3fdacffb890aeebdb0f5a3b6aaa7eeeaf1a38af0a8fe88b9e4b1f61f236d2e64d95733327a62@relay.ultrasound.money")
```

Comment:
Could this example be moved to the Example property of the mevboost [`cobra.Command`](https://pkg.go.dev/github.com/spf13/cobra#Command) instance?